packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md

```diff
@@ -1,12 +1,11 @@
 * **Name**: Enter a name for the Identity Provider configuration.
-* **Client ID**: Paste the app ID or client ID that you obtained when you configured the Identity Provider in the previous section.
-* **Client Secret**: Paste the secret that you obtained in the previous section.
 * **Scopes**: Leave the defaults. These scopes are included when Okta makes an OpenID Connect request to the Identity Provider.
-
     > **Note:** By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory.
-
+* **Client ID**: Paste the app ID or client ID that you obtained when you configured the Identity Provider in the previous section.
+* **Authentication type**: Leave the default of **Client secret** or select **Public key/private key** to automatically generate a public and private key pair. The public key is available for download when you click **Finish**.
+* **Client Secret**: If you are using **Client secret** as the **Authentication type**, paste the secret that you obtained in the previous section.
 * **Authorize requests**: Select **Enable signed requests** to send request parameters to the OpenID provider as an encoded JWT instead of passing the parameters in the URL. <ApiLifecycle access="ea" />
-* **Algorithm**: Select the algorithm to use for the signed requests from the dropdown list. <ApiLifecycle access="ea" />
+* **Algorithm**: Select the algorithm to use for the signed requests from the dropdown list. If you are using **Public key/private key**, you must specify an encryption algorithm, for example: **RSA256**. <ApiLifecycle access="ea" />
```

Comment:
There is a whitespace between this and previous bullet

Reply:
And not encryption. It is Signing Algorithm

Reply:
Unfortunately, it's the EA flag that causes the extra white space. I have made the other updates.